src/Branch.ts

```diff
@@ -504,16 +552,88 @@ export class Branch {
 			sort,
 			start
 		});*/
+
+		const filtersMustSets: string[][] = [];
+		const filtersMustNotSets: string[][] = [];
+		// const filtersShouldSets: string[][] = [];
+		let filtersMustIds: string[] = [];
+		let filtersMustNotIds: string[] = [];
+		// let filtersShouldIds: string[] = [];
+		if (
+			(Array.isArray(filters) && isFilter(filters))
+			|| isFilter(filters)
+		) {
+			const filtersArray = forceArray(filters);
+
+			for (let filtersIndex = 0; filtersIndex < filtersArray.length; filtersIndex++) {
+				const filter = filtersArray[filtersIndex];
+				if (isBooleanFilter(filter)) {
+					const must = (filter as BooleanFilter).boolean.must ? forceArray((filter as BooleanFilter).boolean.must): [];
```

Comment:
```suggestion
					const must = forceArray((filter as BooleanFilter).boolean.must ?? []);
```

Reply:
I guess if the runtime env support it. If we limit which runtime environment we support in package.json, this could be done.